CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ endif ()
 pkg_check_modules(LibZhuyin REQUIRED IMPORTED_TARGET "libzhuyin>=2.3.0")
 
 add_definitions(-DFCITX_GETTEXT_DOMAIN=\"fcitx5-zhuyin\")
+add_definitions(-DDATABASE_FORMAT=\"KyotoCabinet\")
```

Comment:
This is not the right way to work with configure_file
set(DATABASE_FORMAT "KyotoCabinet") will do.

But this is a configure option for libpinyin. This is how libzhuyin detect the option:
https://github.com/libzhuyin/ibus-libzhuyin/blob/9bb4588fd2c59c48e10e799112ef5e7f307a31b2/configure.ac#L67